xml/docu_styleguide.structure.xml

```diff
@@ -3145,6 +3145,66 @@ Entire hard disks are listed as devices without numbers, such as
   </table>
  </sect2>
 
+ <sect2 xml:id="sec-task-element">
+  <title>Task elements</title>
+  <para>
+    The task element encapsulates a procedure, providing an explicit 
+    location for summary information, identifying prerequisites for 
+    the task, examples and pointers to related information. Use it to 
+    present instructions or steps required to perform a specific task.</para>
+    <example xml:id="ex-task-source">
+      <title>Example of a task element (source)</title>
+   <screen>
+&lt;task xml:id="task.example"&gt;
+  &lt;title&gt;An example task&lt;/title&gt;
+  &lt;tasksummary&gt;
+   &lt;para&gt;This is the summary of this task.&lt;/para&gt;
+  &lt;/tasksummary&gt;
+  &lt;taskprerequisites&gt;
+   &lt;para&gt;This contains the requirements.&lt;/para&gt;
+  &lt;/taskprerequisites&gt;
```

Comment:
Should we require a `<title>` for `<tasksummary>` and `<taskprerequisites>`?

Reply:
I don't think so. "Requirements" are rendered as a headline by the style sheets, and the summary does not really need another title.